ext/standard/array.c

```diff
@@ -4220,7 +4220,7 @@ PHP_FUNCTION(array_count_values)
 				ZVAL_LONG(&data, 1);
 				zend_symtable_update(Z_ARRVAL_P(return_value), Z_STR_P(entry), &data);
 			} else {
-				Z_LVAL_P(tmp)++;
+				Z_STR_P(tmp)++;
```

Comment:
This does not look right. The `return_value` array is only supposed to have int values, never strings.